pkg/api/instance.go

```diff
@@ -7,6 +7,13 @@ import (
 	"github.com/longhorn/longhorn-spdk-engine/proto/spdkrpc"
 )
 
+var (
+	dataEngines = map[string]string{
+		"engine_v1": "v1",
+		"engine_v2": "v2",
+	}
+)
```

Comment:
Just use v1/v2 in proto, so don't need to do a conversion here.

Reply:
The conversion is due to https://github.com/longhorn/longhorn-instance-manager/pull/348#discussion_r1437964221